src/printer.js

```diff
@@ -49,18 +49,22 @@ function shouldPrintComma(options, level) {
   }
 }
 
+function hasPrettierIgnoreComment(node) {
```

Comment:
Right, fixed in https://github.com/prettier/prettier/pull/1609/commits/1e60895f2467696876e0fdd27d04e7042776ba3f